src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1428,6 +1429,13 @@ private function getErrorMessages(Exception $e)
                 'Only numbers and decimal points (".") are allowed in the amount fields, e.g. 10.50 or 1050.',
                 'Admin.Orderscustomers.Notification'
             ),
+            InvalidCartRuleDiscountValueException::class => sprintf(
+                '%s<ol><li>%s</li><li>%s</li><li>%s</li></ol>',
+                $this->trans('It looks like the value is invalid:', 'Admin.Orderscustomers.Notification'),
+                $this->trans('Discount value must be greater than 0.', 'Admin.Orderscustomers.Notification'),
```

Comment:
```suggestion
                $this->trans('Percent or amount value must be greater than 0.', 'Admin.Orderscustomers.Notification'),
```

Reply:
cf. https://github.com/PrestaShop/PrestaShop/pull/17852/#issuecomment-604471739

Reply:
@LouiseBonnard my bad, I didn't see you updated [your comment](https://github.com/PrestaShop/PrestaShop/pull/17852#issuecomment-604471739)!

Reply:
No worries, thanks @atomiix!